Makefile

```diff
@@ -48,9 +54,6 @@ help: ## Print help for targets with comments.
 	@echo ""
 	@echo "Useful commands:"
 	@grep -Eh '^[a-zA-Z._-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "  $(shell tput setaf 6 2>/dev/null)%-30s$(shell tput sgr0 2>/dev/null) %s\n", $$1, $$2}'
-	@echo ""
```

Comment:
It doesn't print any variables any more. Let me merge this and send another PR for the bike shedding to avoid folks picking up the broken Makefile.